modules/platforms/cpp/thin-client/include/ignite/impl/thin/transactions/transactions_proxy.h

```diff
@@ -126,7 +127,7 @@ namespace ignite
                             ignite::thin::transactions::TransactionIsolation::Type isolation = ignite::thin::transactions::TransactionIsolation::READ_COMMITTED,
                             int64_t timeout = 0,
                             int32_t txSize = 0,
-                            ignite::common::concurrent::SharedPointer<const char> lbl = "");
+                            ignite::common::concurrent::SharedPointer<ignite::common::FixedSizeArray<char> > lbl = "");
```

Comment:
Why empty string? default should be null string.

Reply:
thanks !